modules/components/pages/inputs/redpanda_common.adoc

```diff
@@ -0,0 +1,284 @@
+
+*Type*: `array`
+
+```yml
+# Examples
+
+topics:
+  - foo
+  - bar
+
+topics:
+  - things.*
+
+topics:
+  - foo,bar
+
+topics:
+  - foo:0
+  - bar:1
+  - bar:3
+
+topics:
+  - foo:0,bar:1,bar:3
+
+topics:
+  - foo:0-5
+```
+
+=== `regexp_topics`
+
+Whether listed topics are interpreted as regular expression patterns for matching multiple topics. When topics are specified with explicit partitions, this field must remain set to `false`.
+
+*Type*: `bool`
+
+*Default*: `false`
+
+=== `rack_id`
+
+A rack specifies where the client is physically located, and changes fetch requests to consume from the closest replica as opposed to the leader replica.
+
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `start_from_oldest`
+
+Whether to consume from the oldest available offset. Otherwise, messages are consumed from the latest offset. This setting is applied when creating a new consumer group or the saved offset no longer exists.
+
+*Type*: `bool`
+
+*Default*: `true`
+
+=== `fetch_max_bytes`
+
+The maximum amount of bytes that a broker tries to send during a fetch. 
+
+If individual records are larger than the `fetch_max_bytes` value, brokers will still send them.
+
+*Type*: `string`
+
+*Default*: `50MiB`
+
+=== `fetch_min_bytes`
+
+The minimum amount of bytes that a broker tries to send during a fetch. This field is equivalent to the Java setting `fetch.min.bytes`.
+
+*Type*: `string`
+
+*Default*: `1B`
+
+=== `fetch_max_partition_bytes`
+
+The maximum amount of bytes that are consumed from a single partition in a fetch request. This field is equivalent to the Java setting `fetch.max.partition.bytes`.
+
+If a single batch is larger than the `fetch_max_partition_bytes` value, the batch is still sent so that the client can make progress. 
+
+*Type*: `string`
+
+*Default*: `1MiB`
+
+=== `consumer_group`
+
+An optional consumer group. When this value is specified: 
+
+- The partitions of any topics, specified in the `topics` field, are automatically distributed across consumers sharing a consumer group
+- Partition offsets are automatically committed and resumed under this name
+
+Consumer groups are not supported when you specify explicit partitions to consume from in the `topics` field.
+
+*Type*: `string`
+
+
+=== `commit_period`
+
+The period of time between each commit of the current partition offsets. Offsets are always committed during shutdown.
+
+
+*Type*: `string`
+
+*Default*: `5s`
+
+=== `partition_buffer_bytes`
+
+A buffer size (in bytes) for each consumed partition, which allows the internal queuing of records before they are flushed. Increasing this value may improve throughput but results in higher memory utilization. 
+
+Each buffer can grow slightly beyond this value.
+
+*Type*: `string`
+
+*Default*: `1MB`
+
+=== `auto_replay_nacks`
```

Comment:
"nacked" is not a word as far as I can tell--it's not in the 3 dictionaries I checked. We should refrain from introducing "non-words" or jargon that is not intuitive to our users. Can this be changed?